packages/cli/src/commands/deploy/latest.ts

```diff
@@ -500,13 +519,13 @@ export default async function main(client, paths, localConfig, args) {
     target = 'production';
   }
 
-  const currentTeam = org.type === 'team' ? org.id : undefined;
+  const currentTeam = org?.type === 'team' ? org.id : undefined;
   const now = new Now({ apiUrl, token, debug: debugEnabled, currentTeam });
   let deployStamp = stamp();
   let deployment = null;
 
   try {
-    const createArgs = {
+    const createArgs: any = {
```

Comment:
I think we could add a type here and use that in createDeploy() too

Reply:
I spent some time trying to properly type this, but the auto-generated types from JS `now.create()` ended up being incompatible. I think I'm going to punt on this for now. We can address in a separate PR.